pipeline/4/Jenkinsfile-tier-0.groovy

```diff
@@ -45,7 +46,13 @@ node(nodeName) {
 			"containerized=false",
 			"addnArgs=--post-results --log-level DEBUG"
 		    ]) {
-			sharedLib.runTestSuite()
+                        catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+                                rc = sharedLib.runTestSuite()
+                                test_results['rpm_sanity'] = rc
+                                if(rc != 0){
+                                        sh "exit 1"
+                                }
+                        }
```

Comment:
Please make the change in `common.groovy`. We need `runTestSuite` not to throw an error